test/framework/resources/aws/utils/nodegroup.go

```diff
@@ -229,6 +230,28 @@ func GetClusterVPCConfig(f *framework.Framework) (*ClusterVPCConfig, error) {
 		PrivateSubnetList: []string{},
 	}
 
+	if len(f.Options.PublicSubnets) > 0 {
+		clusterConfig.PublicSubnetList = strings.Split(f.Options.PublicSubnets, ",")
+	}
+	if len(f.Options.PrivateSubnets) > 0 {
+		clusterConfig.PrivateSubnetList = strings.Split(f.Options.PrivateSubnets, ",")
+	}
+	if len(f.Options.AvailabilityZones) > 0 {
+		clusterConfig.AvailZones = strings.Split(f.Options.AvailabilityZones, ",")
+	}
+	if f.Options.PublicRouteTableID != "" {
+		clusterConfig.PublicRouteTableID = f.Options.PublicRouteTableID
+	}
+
+	// user provided the info so we don't need to look it up
+	if clusterConfig.PublicRouteTableID != "" && len(clusterConfig.PublicSubnetList) > 0 || len(clusterConfig.AvailZones) > 0 {
```

Comment:
nit: can we add some parentheses around the or condition to make this more readable?